packages/core/test/overlay/overlayTests.tsx

```diff
@@ -19,6 +19,7 @@ describe("<Overlay>", () => {
     let wrapper: ReactWrapper<IOverlayProps, any>;
 
     afterEach(() => {
+        document.body.classList.remove(Classes.OVERLAY_OPEN);
```

Comment:
Ah, yes, I noticed that when I was fixing other tests but didn't come back to omit it here. Done. 👍